<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewer</class>
 <widget class="QMainWindow" name="ImageViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор изображений</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="imgView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Опции</string>
    </property>
    <widget class="QMenu" name="menuset_frame">
     <property name="title">
      <string>сделать рамку</string>
     </property>
     <addaction name="separator"/>
     <addaction name="separator"/>
     <addaction name="single_color"/>
     <addaction name="double_color"/>
    </widget>
    <addaction name="menuset_frame"/>
    <addaction name="actionsearch_rectangles"/>
    <addaction name="actionchange_color"/>
    <addaction name="actioncopy_frame"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="single_color">
   <property name="text">
    <string>один цвет</string>
   </property>
  </action>
  <action name="double_color">
   <property name="text">
    <string>два цвета</string>
   </property>
  </action>
  <action name="actionsearch_rectangles">
   <property name="text">
    <string>найти прямоугольники</string>
   </property>
  </action>
  <action name="actionchange_color">
   <property name="text">
    <string>заменить цвет</string>
   </property>
  </action>
  <action name="actioncopy_frame">
   <property name="text">
    <string>копировать область</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
